rep-0144.rst

```diff
@@ -31,42 +31,55 @@ Some of those rules are mandatory, others merely advised.
 Package Naming
 ==============
 
-Mandatory Rules
----------------
-
 By habit, a package name is often used as a namespace (in C++ or any other language).
 Thus, the naming rules have to be strict.
 
+Definitions
+-----------
+
 * ``alphanumerics`` are ``a-z0-9`` only
 * ``alphabetics`` are ``a-z`` only
-* it must only consist of alphanumerics and ``_`` separators.
-  Other symbols might not be supported by some OSes (e.g. unicode characters) or would
-  make it hard to follow OSes conventions/
-* they must be at least two characters long and must start with an alphabetic character.
+
+For clarity these are both specifically exclusively lowercase.
```

Comment:
I don't think "clarity" is a good reason for this. Namespaces usually map to directory names and those might not support case sensitive names depending on the filesystem. Since names which only differ in case would collide in that scenario lower case is is being used exclusively.

Reply:
This statement isn't giving a reason, it's clarifying that `a-z` is specified specifically to not include `A-Z`. I can add the additional reason for the lowercase in the rules below.